QueryCiv3/BiqSections/Unit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct UNIT
+    {
+        public int Length;
+
+        private fixed byte Text[32];
+        public string Name { get => Util.GetString(ref this, 4, 32); }
+
+        public int OwnerType; // 0: None, 1: Barbarian, 2: Civ, 3: Player
+        public int ExperienceLevel;
+        public int Owner;
+        public int Unit;
```

Comment:
This should be UnitType, most likely.  It's a PRTO reference, but with just Unit as the name, it's ambiguous whether it's a PRTO reference or perhaps a reference to another UNIT type (PRTO/UNIT can be confusing in general in Civ3 since they're both generically referred to as "unit" in different contexts).